tests/migration_test.go

```diff
@@ -1209,6 +1211,11 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				By("Checking that the VirtualMachineInstance console has expected output")
 				Expect(libnet.WithIPv6(console.LoginToFedora)(vmi)).To(Succeed(), "Should be able to login to the Fedora VM")
 
+				if migrationConfiguration != nil {
+					By("Running stress test to allow transition to post-copy")
+					runStressTest(vmi, "800", stressdefaultTimeout)
```

Comment:
It is the third time that this "800" literal is used, could we define a constant for it?

Reply:
You're absolutely right. Thanks, it's added now.